.devcontainer/devcontainer.json

```
@@ -6,12 +6,12 @@
         "ghcr.io/devcontainers/features/go:1": {},
         "ghcr.io/azutake/devcontainer-features/go-packages-install:0": {},
         "ghcr.io/devcontainers/features/docker-in-docker:2": {},
-        "ghcr.io/devcontainers-contrib/features/gh-release:1.0.24": {
-            "repo": "buildpacks/pack",
+        "./pack-cli": {
             "version": "v0.37.0",
-            "binaryNames": "pack",
         },
```

It looks like there's no need for such a radical change for `gh-release`:

```suggestion
        "ghcr.io/devcontainers-extra/features/gh-release:1.0.26": {
            "repo": "buildpacks/pack",
            "version": "v0.37.0",
            "binaryNames": "pack",
        },
```
should be enough.